### ISSUE TITLE
The diagram of the exchanges is mandating a pre-configuration between each pair of AS/RS

### ISSUE THREAD

**Issue by Denisthemalice** [ietf-wg-gnap/gnap-core-protocol#285]
On page 10, the first exchange is now done with an AS instead of a RS. This means that the document is “AS centric” 
whereas it should be “RS centric”.

The (A) transaction is described as: 

> (A) The end-user interacts with the client instance to indicate a need for resources on behalf of the RO.

However, the end-user has no clue to know who the RO may be. It only knows which resource it wants to access to and which operation he is willing to perform on that resource. At this time, it does not even know which AS(s) is (are) trusted by the RS hosting this resource.

The step (1) is no more supported by a transaction as it was originally the case in earlier versions but is some kind of internal step where the client instance by some kind of magic determines “what access is needed and which AS to approach for access”.

The document is currently assuming a pre-configuration between each pair of AS/RS. This is not a scalable approach.
 Other cases, where an AS does not need any pre-relationship with a RS should be supported.

**Comment by jricher** [comment]
This is not true. The relationship between AS and RS can be dynamic as discussed in the RS document. That relationship is out of scope for the core protocol which is concerned with the client instance.

**Comment by fimbault** [comment]
We also have section 9.1 "RS-first Method of AS Discovery" which permits a scalable approach.
